# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../tlsf_alloc/makeinstall.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2008-06-01 22:57:45.000000000 +0000
+++ ./Makefile	2008-06-01 23:16:26.000000000 +0000
@@ -6,6 +6,10 @@
 CFLAGS+= -DUSE_MMAP=1
 CFLAGS+= -DUSE_SBRK=1
 
+DESTDIR=
+libdir=/usr/lib
+includedir=/usr/include
+
 # CFLAGS+=-ftest-coverage -fprofile-arcs
 CC=gcc
 
@@ -19,3 +23,6 @@
 indent:
 	indent -kr -l120 -br -ce -npsl -nut -i4 *.[hc]
 
+install: tlsf.o
+	cp -vf tlsf.o $(DESTDIR)$(libdir)
+	cp -vf tlsf.h $(DESTDIR)$(includedir)
